v2/emailpassword/testing/testing-with-postman.mdx

```diff
@@ -60,34 +66,34 @@ app.post("/change-user-data", Session.verifySession(), async (req, res) => {
 })
 
 ```
+
 </TabItem>
 </BackendSDKTabs>
 
 - In Postman, set the request type to `POST`.
 - Set the URL to `http://localhost:3001/change-user-data`
-- In the Header tab, set key `rid` with value `emailpassword`.
+- In the Header tab, set key `rid` with value `^{ridRecipeName}`.
 - If you have the `antiCsrf` attribute set to `VIA_TOKEN` in your backend SuperTokens config, then, in the Postman Header tab, set the key as `anti-csrf` and value as the `anti-csrf` token retrieved from the login response.
-- On a successful response, the response body will contain the `user` object
+- On a successful response, the response body will contain user data
 
 :::important
 By default, for `GET` APIs, you don't need to provide the `anti-csrf` request header as anti-CSRF checks are only done in non-`GET` APIs
 :::
 
-<img width="600px" src="/img/emailpassword/st-ep-change-user.png" />
+<img width="600px" src="/img/change-user.png" />
```

Comment:
You can use variables in the src link as well.. So use those and have different images as expected.

Reply:
done